back-office/international/translations/translations.md

```diff
@@ -0,0 +1,140 @@
+- Admin.AdvParameters.Help
+- Admin.AdvParameters.Notification
+- Admin.Global
+
+Only the leaves of this tree are clickable. On the right, the list of  wordings belonging to the currently selected domain are shown, each followed by a text field containing its translation to the selected language.
+
+The user can edit translations by filling out the fields, and save them.
+
+On the top of the page, there's a search box. When the user searches for a wording, the list is filtered, meaning that wordings not matching the search expression are no longer shown on the list and domains containing no matching wording are grayed out.
+
+#### Translation sources & storage
+
+This section covers where wordings are sourced from and where translations are stored to according to the translation type chosen.
+
+##### Generalities
+
+Usually and unless specified otherwise:
+
+- Wordings are sourced from XLF files stored somewhere, one file per translation domain.
+- User-edited translations are stored in the database
+
+##### Back office translations
+
+Wordings are sourced from the Core's _default catalogue_, which is stored as physical XLF files (one per translation domain) in `app/Resources/translations/default`. This catalogue is to be filtered by keeping only the translation domains whose name starts with **"Admin"**.
+
+Translations are retrieved from two sources: 
+
+1. The core: XLF translation files for the chosen language, stored in `app/Resources/translations/{locale_code}`, one per domain.
+2. The database: if translations were customized previously through this interface.
+
+If the same wording is translated in both core files and database, the database translation is shown.
+
+##### Front office translations
+
+This translation type has two different subcases:
+
+**i. If the user chooses "Core translations"**
+
+The behavior is exactly like "back office translations", only that translation domains are filtered by domains starting with **"Shop"** instead.
+
+**ii. If the user chooses a theme.**
+
+In this case, wordings are sourced by analyzing all the theme's files. Since this process is slow, the result of this analysis is stored as XLF files in local filesystem cache. If the cache is found, then wordings are read from there. The user can refresh this cache by clicking on the "Find new wordings" button at the top of the page.
+
+Translations are sourced from XLF files within the theme in its `translations/{locale_code}` directory, one per domain. These files may or may not be present.
+
+User-edited translations are stored in database. To avoid collisions, theme translations must be kept associated to the theme they belong. If a theme redefines a Core translation (ie. same domain, same wording), then the redefined translation should only be used when the FO is using that theme (eg. in case of multishop with different themes for each shop, each theme should use the its own translation even if they both use the same given wording/domain pair).
```

Comment:
```suggestion
User-edited translations are stored in database. To avoid collisions, theme translations must be kept associated to the theme they belong. If a theme redefines a Core translation (ie. same domain, same wording), then the redefined translation should only be used when the FO is using that theme (eg. in case of multishop with different themes for each shop, each theme should use the its own translation even if they both use the same given wording/domain pair). Redefined core translations should only be saved if they're different than the ones defined when not in the theme's context.
```